setup.py

```diff
@@ -45,12 +45,11 @@
             "sphinx_rtd_theme",
             "recommonmark",
             "open-simulation-interface",
-            "doxygen-interface",
             "defusedxml",
             "colorama",
             "tabulate",
             "progress",
-            "protobuf==3.9.1",
+            "protobuf",
```

Comment:
Die Begründung (aus unserer internen Kommunikation):
Es sieht so aus, als hätte der osi-validator da ein größeres Problem mit Protobuf3: viele der Checks prüfen, ob ein Feld gesetzt ist, aber in proto3 kann man gar nicht mehr feststellen, ob eine Feld nicht gesetzt wurde, oder den Default-Wert hat.

Beispiel: 

TrafficSign.MainSign.Classification.vertically_mirrored.is_set(None) does not comply in SensorView.global_ground_truth.traffic_sign.main_sign.classification

vertically_mirrored ist ein bool. Wenn der Wert bei der Generierung auf false gesetzt wird (der Default), dann wird er in proto3 nicht übertragen.

Damit ist ein größerer Teil der osi-validator checks im Zusammenhang mit proto3 hinfällig.